### ISSUE TITLE
How can I tell when the drawer has closed?

### ISSUE THREAD

**Issue by infiniteNIL** [erichoracek/MSDynamicsDrawerViewController#45]
I want to know when the drawer has closed, but I don't see a way to do that. I searched the issues and it seems like the issue came up before but when I search for the code those discussions talked about, I don't find anything.

**Comment by fcanas** [comment]
There's a solution in progress to add delegation to the drawer view controller (see https://github.com/fcanas/MSDynamicsDrawerViewController). I hope to have it finished within a few days. Do you think it will fit your needs?

```
- (void)dynamicsDrawerViewController:(MSDynamicsDrawerViewController *)drawerViewController willUpdateToPaneState:(MSDynamicsDrawerPaneState)paneState forDirection:(MSDynamicsDrawerDirection)direction;

- (void)dynamicsDrawerViewController:(MSDynamicsDrawerViewController *)drawerViewController didUpdateToPaneState:(MSDynamicsDrawerPaneState)paneState forDirection:(MSDynamicsDrawerDirection)direction;

- (void)dynamicsDrawerViewControllerDidFinishAnimating:(MSDynamicsDrawerViewController *)drawerViewController;
```

**Comment by infiniteNIL** [comment]
Yes, that will work.
-- 
Rod Schmidt
infiniteNIL Software

On December 12, 2013 at 11:17:39 AM, Fabian (notifications@github.com) wrote:

There's a solution in progress to add delegation to the drawer view controller (see https://github.com/fcanas/MSDynamicsDrawerViewController). I hope to have it finished within a few days. Do you think it will fit your needs?
- (void)dynamicsDrawerViewController:(MSDynamicsDrawerViewController *)drawerViewController willUpdateToPaneState:(MSDynamicsDrawerPaneState)paneState forDirection:(MSDynamicsDrawerDirection)direction;
- (void)dynamicsDrawerViewController:(MSDynamicsDrawerViewController *)drawerViewController didUpdateToPaneState:(MSDynamicsDrawerPaneState)paneState forDirection:(MSDynamicsDrawerDirection)direction;
- (void)dynamicsDrawerViewControllerDidFinishAnimating:(MSDynamicsDrawerViewController *)drawerViewController;

**Comment by erichoracek** [comment]
Should be fixed in #59. Let me know if you have any feedback or if this implementation doesn't fit your use case. Thanks!
